test/functional/apps/dashboard/embeddable_data_grid.ts

```diff
@@ -47,12 +47,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('are added when a cell filter is clicked', async function () {
-      await find.clickByCssSelector(`[role="gridcell"]:nth-child(3)`);
+      await find.clickByCssSelector(`[role="gridcell"]:nth-child(4)`);
       // needs a short delay between becoming visible & being clickable
       await PageObjects.common.sleep(250);
       await find.clickByCssSelector(`[data-test-subj="filterOutButton"]`);
       await PageObjects.header.waitUntilLoadingHasFinished();
-      await find.clickByCssSelector(`[role="gridcell"]:nth-child(3)`);
+      await find.clickByCssSelector(`[role="gridcell"]:nth-child(4)`);
```

Comment:
dear presentation team, since there is now 1 cell more , this value had to be increased